--- digikam-5.1.0.orig/doc-translated/digikam/it/CMakeLists.txt
+++ digikam-5.1.0/doc-translated/digikam/it/CMakeLists.txt
@@ -1 +1,2 @@
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/it/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/it/ SUBDIR digikam )
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/it/ SUBDIR showfoto )
--- digikam-5.1.0.orig/doc-translated/digikam/nl/CMakeLists.txt
+++ digikam-5.1.0/doc-translated/digikam/nl/CMakeLists.txt
@@ -1 +1,2 @@
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ SUBDIR digikam )
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ SUBDIR showfoto )
--- digikam-5.1.0.orig/doc-translated/digikam/pt/CMakeLists.txt
+++ digikam-5.1.0/doc-translated/digikam/pt/CMakeLists.txt
@@ -1 +1,2 @@
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt/ SUBDIR digikam )
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt/ SUBDIR showfoto )
--- digikam-5.1.0.orig/doc-translated/digikam/pt_BR/CMakeLists.txt
+++ digikam-5.1.0/doc-translated/digikam/pt_BR/CMakeLists.txt
@@ -1 +1,2 @@
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt_BR/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt_BR/ SUBDIR digikam )
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt_BR/ SUBDIR showfoto )
--- digikam-5.1.0.orig/doc-translated/digikam/sv/CMakeLists.txt
+++ digikam-5.1.0/doc-translated/digikam/sv/CMakeLists.txt
@@ -1 +1,2 @@
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/sv/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/sv/ SUBDIR digikam )
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/sv/ SUBDIR showfoto )
--- digikam-5.1.0.orig/doc-translated/digikam/uk/CMakeLists.txt
+++ digikam-5.1.0/doc-translated/digikam/uk/CMakeLists.txt
@@ -1 +1,2 @@
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ SUBDIR digikam )
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ SUBDIR showfoto )
