Summary: Fix linkage within documentation.
Author: Steve M. Robbins <smr@debian.org>
Bug: 749684
Forwarded: no



--- digikam-4.14.0.orig/doc/digikam/digikam/menu-descriptions.docbook
+++ digikam-4.14.0/doc/digikam/digikam/menu-descriptions.docbook
@@ -296,7 +296,7 @@
                 </action></para>
                 <para>
                     See the dedicated 
-                    <ulink url="help:/kipi-plugins/geolocalization.html">Geolocation manual</ulink> 
+                    <ulink url="help:/kipi-plugins/geolocation.html">Geolocation manual</ulink> 
                     for more information.
                 </para>     
             </listitem>
@@ -310,11 +310,6 @@
                 <para><action>
                     Kipi-plugin for adjusting time and date of the currently selected images. 
                 </action></para>
-                <para>
-                    See the dedicated 
-                    <ulink url="help:/kipi-plugins/timeadjust.html">Time Adjust manual</ulink> 
-                    for more information.
-                </para>     
             </listitem>
         </varlistentry>
         
@@ -328,7 +323,7 @@
                 </action></para>
                 <para>
                     See the dedicated 
-                    <ulink url="help:/kipi-plugins/metadata.html">Metadata manual</ulink> 
+                    <ulink url="help:/kipi-plugins/metadataeditor.html">Metadata manual</ulink> 
                     for more information.
                 </para>     
             </listitem>
@@ -635,11 +630,6 @@
                 <para><action>
                     Kipi-plugin to find duplicate images in Albums Database.
                 </action></para>
-                <para>
-                    See the dedicated 
-                    <ulink url="help:/kipi-plugins/findduplicateimages.html">Find Duplicate Images manual</ulink> 
-                    for more information.
-                </para>     
             </listitem>            
         </varlistentry>
         
@@ -693,11 +683,6 @@
                 <para><action>
                     Kipi-plugin for creating a calendar with Album items.
                 </action></para>
-                <para>
-                    See the dedicated 
-                    <ulink url="help:/kipi-plugins/calendar.html">Calendar manual</ulink> 
-                    for more information.
-                </para>     
             </listitem>            
         </varlistentry>
 
@@ -709,11 +694,6 @@
                 <para><action>
                     Kipi-plugin OpenGL full screen image viewer with the selected image.
                 </action></para>
-                <para>
-                    See the dedicated 
-                    <ulink url="help:/kipi-plugins/openglviewer.html">OpenGL viewer manual</ulink> 
-                    for more information.
-                </para>     
             </listitem>            
         </varlistentry>
         
@@ -725,11 +705,6 @@
                 <para><action>
                     Kipi-plugin to set up a list of remote galleries for exporting.
                 </action></para>
-                <para>
-                    See the dedicated 
-                    <ulink url="help:/kipi-plugins/remotegalleries.html">Remote galleries manual</ulink> 
-                    for more information.
-                </para>     
             </listitem>
         </varlistentry>
     
@@ -899,11 +874,6 @@
                 <para><action>
                     Kipi-plugin to archive Albums to &CD; or DVD using K3b.
                 </action></para>
-                <para>
-                    See the dedicated 
-                    <ulink url="help:/kipi-plugins/cdarchiving.html">&CD; Archiving manual</ulink> 
-                    for more information.
-                </para>     
             </listitem>
         </varlistentry>
         
@@ -915,11 +885,6 @@
                 <para><action>
                     Kipi-plugin that exports the images series into a KML structure. KML files can then be opened and displayed with GoogleEarth.
                 </action></para>
-                <para>
-                    See the dedicated 
-                    <ulink url="help:/kipi-plugins/fkmlrexport.html">KML Export manual</ulink> 
-                    for more information.
-                </para>
             </listitem>
         </varlistentry>
         
@@ -963,11 +928,6 @@
                 <para><action>
                     Kipi-plugin to upload or update images to <ulink url="http://www.gallery2.org/">web galleries</ulink>.
                 </action></para>
-                <para>
-                    See the dedicated 
-                    <ulink url="help:/kipi-plugins/galleryexport.html">Gallery Export manual</ulink> 
-                    for more information.
-                </para>     
             </listitem>
         </varlistentry>
         
@@ -979,11 +939,6 @@
                 <para><action>
                     Kipi-plugin upload or update images to Googles <ulink url="http://picasaweb.google.com/">Picasaweb galleries</ulink>.
                 </action></para>
-                <para>
-                    See the dedicated 
-                    <ulink url="help:/kipi-plugins/picasaweb.html">Picasaweb export manual</ulink> 
-                    for more information.
-                </para>     
             </listitem>
         </varlistentry>
         
