--- digikam-4.5.0.orig/core/CMakeLists.txt
+++ digikam-4.5.0/core/CMakeLists.txt
@@ -223,17 +223,7 @@ if(ENABLE_INTERNALMYSQL)
 
     # Check MySQL executables
 
-    find_program(SERVERCMD_MYSQL mysqld
-                 PATHS /usr/sbin
-                       /usr/local/sbin
-                       /usr/libexec
-                       /usr/local/libexec
-                       /opt/local/libexec
-                       /opt/mysql/libexec
-                       /usr/mysql/bin
-                       /opt/mysql/sbin
-                       ${MYSQLD_PATH}
-                 ENV MYSQLD_PATH )
+    SET( SERVERCMD_MYSQL /usr/sbin/mysqld-digikam )
     message(STATUS "Found MySQL server executable at: ${SERVERCMD_MYSQL}")
 
     # NOTE: under windows, mysql_install_db PERL script is named "mysql_install_db.pl"
