--- a/extra/kipi-plugins/sendimages/sendimages.cpp
+++ b/extra/kipi-plugins/sendimages/sendimages.cpp
@@ -593,6 +593,7 @@
 
                 case EmailSettings::NETSCAPE:
                 case EmailSettings::THUNDERBIRD:
+                case EmailSettings::ICEDOVE:
                 case EmailSettings::GMAILAGENT:
                 {
                     QString prog;
@@ -605,6 +606,10 @@
                     {
                         prog = QString("thunderbird");
                     }
+                    else if (d->settings.emailProgram == EmailSettings::ICEDOVE)
+                    {
+                        prog = QString("icedove");
+                    }
                     else
                     {
                         prog = QString("gmailagent");
--- a/extra/kipi-plugins/sendimages/emailsettings.h
+++ b/extra/kipi-plugins/sendimages/emailsettings.h
@@ -70,6 +70,7 @@
         NETSCAPE,
         SYLPHEED,
         SYLPHEEDCLAWS,
+        ICEDOVE,
         THUNDERBIRD
     };
 
--- a/extra/kipi-plugins/sendimages/settingswidget.cpp
+++ b/extra/kipi-plugins/sendimages/settingswidget.cpp
@@ -100,6 +100,7 @@
     d->mailAgentName->insertItem(EmailSettings::SYLPHEED,      "Sylpheed");
     d->mailAgentName->insertItem(EmailSettings::SYLPHEEDCLAWS, "Sylpheed-Claws");
     d->mailAgentName->insertItem(EmailSettings::THUNDERBIRD,   "Thunderbird");
+    d->mailAgentName->insertItem(EmailSettings::ICEDOVE,       "Icedove");
     d->mailAgentName->setCurrentIndex(EmailSettings::DEFAULT);
     d->mailAgentName->setWhatsThis(i18n("Select your preferred external email program here. "
                                         "<b>Default</b> is the current email program set in KDE "
