Index: digikam-4.0.0~beta1/extra/kipi-plugins/sendimages/sendimages.cpp
===================================================================
--- digikam-4.0.0~beta1.orig/extra/kipi-plugins/sendimages/sendimages.cpp	2013-12-02 21:24:24.000000000 +1100
+++ digikam-4.0.0~beta1/extra/kipi-plugins/sendimages/sendimages.cpp	2013-12-11 22:33:28.257866860 +1100
@@ -583,6 +583,7 @@
 
                 case EmailSettings::NETSCAPE:
                 case EmailSettings::THUNDERBIRD:
+                case EmailSettings::ICEDOVE:
                 case EmailSettings::GMAILAGENT:
                 {
                     QString prog;
@@ -595,6 +596,10 @@
                     {
                         prog = QString("thunderbird");
                     }
+                    else if (d->settings.emailProgram == EmailSettings::ICEDOVE)
+                    {
+                        prog = QString("icedove");
+                    }
                     else
                     {
                         prog = QString("gmailagent");
Index: digikam-4.0.0~beta1/extra/kipi-plugins/sendimages/emailsettings.h
===================================================================
--- digikam-4.0.0~beta1.orig/extra/kipi-plugins/sendimages/emailsettings.h	2013-12-02 21:24:24.000000000 +1100
+++ digikam-4.0.0~beta1/extra/kipi-plugins/sendimages/emailsettings.h	2013-12-11 22:32:19.483052140 +1100
@@ -70,6 +70,7 @@
         NETSCAPE,
         SYLPHEED,
         SYLPHEEDCLAWS,
+        ICEDOVE,
         THUNDERBIRD
     };
 
Index: digikam-4.0.0~beta1/extra/kipi-plugins/sendimages/settingswidget.cpp
===================================================================
--- digikam-4.0.0~beta1.orig/extra/kipi-plugins/sendimages/settingswidget.cpp	2013-12-02 21:24:24.000000000 +1100
+++ digikam-4.0.0~beta1/extra/kipi-plugins/sendimages/settingswidget.cpp	2013-12-11 22:32:19.483052140 +1100
@@ -100,6 +100,7 @@
     d->mailAgentName->insertItem(EmailSettings::SYLPHEED,      "Sylpheed");
     d->mailAgentName->insertItem(EmailSettings::SYLPHEEDCLAWS, "Sylpheed-Claws");
     d->mailAgentName->insertItem(EmailSettings::THUNDERBIRD,   "Thunderbird");
+    d->mailAgentName->insertItem(EmailSettings::ICEDOVE,       "Icedove");
     d->mailAgentName->setCurrentIndex(EmailSettings::DEFAULT);
     d->mailAgentName->setWhatsThis(i18n("Select your preferred external email program here. "
                                         "<b>Default</b> is the current email program set in KDE "
